FrontendApartmentReservation/src/components/Pages/SignUpPage/SignUpPage.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as actionCreators from '../../../actions';
+
+class SignUpPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            inputName: '',
+            inputSurname: '',
+            inputEmail: '',
+            inputPassword: '',
+            inputOffice: '',
+        };
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleNameChange = this.handleNameChange.bind(this);
+        this.handleSurnameChange = this.handleSurnameChange.bind(this);
+        this.handleEmailChange = this.handleEmailChange.bind(this);
+        this.handlePasswordChange = this.handlePasswordChange.bind(this);
+        this.handleOfficeChange = this.handleOfficeChange.bind(this);
+
+    }
+
+    handleSubmit(e) {
+        e.preventDefault();
+        const {inputName, inputSurname, inputEmail, inputPassword, inputOffice} = this.state;
+        const regex = new RegExp("^(?=.*[a-z])(?=.*[A-Z])(?=.*\d).{5,15}$");
```

Comment:
Why is it `{5,15}`?

Reply:
I think it's the same regex as from my backend code